include/libtorrent/torrent_info.hpp

```diff
@@ -418,13 +418,15 @@ namespace libtorrent
 		// to peers over anything other than the i2p network.
 		bool is_i2p() const { return (m_flags & i2p) != 0; }
 
+		int piece_size(piece_index_t index) const { return m_files.piece_size(index); }
+
 		// ``hash_for_piece()`` takes a piece-index and returns the 20-bytes
 		// sha1-hash for that piece and ``info_hash()`` returns the 20-bytes
 		// sha1-hash for the info-section of the torrent file.
+		sha1_hash hash_for_piece(piece_index_t index) const;
```

Comment:
the two ways of getting a hash for a piece I think should be grouped together. I suppose ``piece_size`` could have it's own entry